db.js

```diff
@@ -113,7 +113,7 @@ exports.init = function (sbot, config) {
     )
   }
 
-  function add(msg, cb) {
+  function rawAdd(msg, cb) {
     const guard = guardAgainstDuplicateLogs('add()')
```

Comment:
Because `rawAdd` is an internal method, it doesn't make sense to have `guardAgainstDuplicateLogs` anymore, could you remove it? :)

Reply:
Thanks, yes!